must-gather/collection-scripts/gather_clusterscoped_resources

```diff
@@ -67,3 +67,18 @@ for command_desc in "${commands_desc[@]}"; do
      COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/cluster-scoped-resources/oc_output/desc_${command_desc// /_}
     { oc describe "${command_desc}"; } >> "${COMMAND_OUTPUT_FILE}"
 done
+
+#Add ocs ready nodes to the list
+nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/\yReady\y/{print $1}')
+
+#collecting journal logs
+for node in ${nodes}; do
+    printf "collecting crash and journal logs  from node %s \n"  "${node}"
+    oc debug nodes/"${node}" << CMDS
+       chroot /host
+       cd var
```

Comment:
do cd var/logs or do it in PWD.  Not a big issue if you want to do from var good to just mention it in the comment why var.

Reply:
Did `var/log`